ios/PresentationLayer/SampleComposeMultiplatform/Sources/SampleComposeMultiplatform/PlatformSpecificView/CheckboxView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Created by Julia Jakubcova on 25/11/2024
+//  Copyright © 2024 Matee. All rights reserved.
+//
+
+import SwiftUI
+
+struct CheckboxView: View {
+    
+    private var text: String
+    private var isChecked: Bool
+    private var onCheckedChange: (Bool) -> Void
+    
+    init(text: String, isChecked: Bool, onCheckedChange: @escaping (Bool) -> Void) {
+        self.text = text
+        self.isChecked = isChecked
+        self.onCheckedChange = onCheckedChange
+    }
+    
+    var body: some View {
+        HStack {
+            Image(systemSymbol: isChecked ? .checkmarkSquareFill : .square)
```

Comment:
Is this really how they do checkboxes on iOS? 😄

Reply:
I'm afraid so, the other way is to modify a switch (toggle button) 😄